license_copyright/license.yml

```
@@ -0,0 +1,26 @@
+name: 'Update copyright license'
+on:
+  schedule:
+    - cron: '0 0 1 1 *'
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Update the License
+        run: |
+          (echo "Copyright (C) 2016-$(date +"%Y") TheAlgorithms and contributors";  tail -n +2 License) > License.tmp
+          mv License.tmp License
```

```suggestion
        run: |
          (echo "Copyright (C) 2016-$(date +"%Y") TheAlgorithms and contributors";  tail -n +2 LICENSE) > License.tmp
          mv License.tmp LICENSE
```

However, since you're making this more general as an action - I would recommend to make this a variable instead of `License` or `LICENSE` since the license file can be named in myriad of ways.

---

Some repositories also use `LICENSE.txt` or `LICENSE.md` (that's what I use).